ts/containers/staking/header/header.ts

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { Dispatch } from 'redux';
+import { State } from 'ts/redux/reducer';
+
+import { Header as HeaderComponent } from 'ts/components/staking/header/header';
+import { Dispatcher } from 'ts/redux/dispatcher';
+
+interface IHeaderProps {
+    location?: Location;
+    isNavToggled?: boolean;
+    toggleMobileNav?: () => void;
+}
+
+interface IConnectedDispatch {
+    dispatcher: Dispatcher;
+}
+
+interface ConnectedState {}
```

Comment:
In the future this will need to know if a wallet is connected for the indicator on the mobile menu